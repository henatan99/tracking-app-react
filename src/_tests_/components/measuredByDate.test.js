import React from 'react';
import { create } from 'react-test-renderer';
import Enzyme from 'enzyme';
import { shallow, mount } from 'enzyme';
import { MemoryRouter } from 'react-router-dom';
import Adapter from 'enzyme-adapter-react-16';

import MeasuredByDate from '../../components/measuredByDate';


Enzyme.configure({ adapter: new Adapter() });

let wrapper;

beforeEach(() => {
  wrapper = shallow(<MeasuredByDate />);
});

describe('FooterNav', () => {
  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });

//   it('should render four <li>', () => {
//     const wrapper = mount(
//       <MemoryRouter>
//         <FooterNav />
//       </MemoryRouter>
//     );
//     expect(wrapper.find('li').length).toBe(4);
//   });
});
    