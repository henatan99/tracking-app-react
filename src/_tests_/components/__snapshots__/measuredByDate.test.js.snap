// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterNav renders correctly 1`] = `ShallowWrapper {}`;
